app/templates/components/schema-field-radio.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="form-group">
+  {{#if property.title}}
+    <label for="{{key}}">{{property.title}}</label>
+  {{/if}}
+  <div class="radio">
+    <div>
+      {{#radio-button value=false groupValue=value name=key changed="changed"}}
```

Comment:
> Pass two properties to the component: value and groupValue. A radio-button will be in a checked state when groupValue === value.

Woah! You are obviously correct, but this API is definitely confusing (considering that `group` is a thing already in a `<radio>` element.